test/resources/stubs/module-definition-test.json

```diff
@@ -341,7 +347,62 @@
         "1.1.1/MD-2_M-2_MI-1_O-2-2_R-3",
         "1.1.2/MD-2_M-1_MI-1_O-2-2_R-3"
       ],
-      "description": ""
+      "description": "",
+      "comment": ""
+    }
+  },
+  "group_ranges": {
+    "0": {
+      "name": "Heating",
+      "children": {
+        "0/0": {
+          "name": "Wohnzimmer",
+          "children": {},
+          "group_adresses": [
+            "0/0/1",
+            "0/0/2",
+            "0/0/3"
+          ],
+          "address_start": "0/0/1",
+          "address_end": "0/0/255",
+          "comment": ""
+        },
+        "0/1": {
+          "name": "Kitchen",
+          "children": {},
+          "group_adresses": [
+            "0/1/2",
+            "0/1/1"
+          ],
+          "address_start": "0/1/0",
+          "address_end": "0/1/255",
+          "comment": ""
+        },
+        "0/2": {
+          "name": "Bathroom",
+          "children": {},
+          "group_adresses": [
+            "0/2/2"
+          ],
+          "address_start": "0/2/0",
+          "address_end": "0/2/255",
+          "comment": ""
+        },
+        "0/7": {
+          "name": "Common",
+          "children": {},
+          "group_adresses": [
+            "0/7/2"
+          ],
+          "address_start": "0/7/0",
+          "address_end": "0/7/255",
+          "comment": ""
+        }
+      },
+      "group_adresses": [],
```

Comment:
On 3-level this is always an empty list - same as above for `children` in middle groups.

I wonder if it would be a good idea to use different schemas for main group and middle group (and only use the middle group schema for 2-level).